tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "noEmit": true,
+    "baseUrl": ".",
+    "paths": {
+      "classwrap": ["src", "classwrap.d.ts"]
```

Comment:
Here to use the `classwrap` module locally without the need of the `npm link` trick